metabase/metabase
# Add test to dashboard parameter should occupy whole horizon space when navigating to dashboards inapp in interactive embed

Closes https://github.com/metabase/metabase/issues/30645

### Description

Add an E2E test to cover #30646. Basically, this issue has been fixed since *.46.7 please read https://github.com/metabase/metabase/issues/30645#issuecomment-1683788545

I've also try running this test against 0.46.1, and 0.46.7


### How to verify

The easiest way to test this is:
1. Modify this function https://github.com/metabase/metabase/blob/f9d328cb8f597e078594da9e374d01ed3cb2df4d/frontend/src/metabase/lib/dom.js#L14-L20
    to return `true`.
2. Create a dashboard
3. Add a dashboard filter, could be any filter
4. Add a text card and makes it super long.
5. Save the dashboard
6. Go to the home page `/` (*The original issue also says to include certain params, but I found that unnecessary since I could reproduce without those parameters.*)
7. Navigate to the newly created dashboard e.g. via the sidebar or the by the search bar
8. Scroll to the bottom.

You shouldn't see the visual problem as reported in the issue.

### Demo

#### 0.46.1 (where the issue was reported)
![0 46 1 test](https://github.com/metabase/metabase/assets/1937582/3aacfe12-8012-4a50-8478-3aef80e98f4e)

#### 0.46.7 (where the fix was landed)
![0 46 7 test](https://github.com/metabase/metabase/assets/1937582/7998837c-e2ec-4966-a28f-fb32fcf5b273)

#### master@9c4abbb8dddba5f41c418df698d11258d9b08b05
![master@9c4abbb8dddba5f41c418df698d11258d9b08b05](https://github.com/metabase/metabase/assets/1937582/9478149d-50dd-4302-afa8-2b4c4347ea79)


### Checklist
